modules/nw-http2-haproxy.adoc

```diff
@@ -6,31 +6,37 @@
 [id="nw-http2-haproxy_{context}"]
 = Enabling HTTP/2 Ingress connectivity
 
-You can enable transparent end-to-end HTTP/2 connectivity in HAProxy. It allows application owners to make use of HTTP/2 protocol capabilities, including single connection, header compression, binary streams, and more.
+You can enable transparent end-to-end HTTP/2 connectivity in HAProxy, so that an application owner can access HTTP/2 protocol capabilities, which includes single connection, header compression, binary streams, and other features.
 
-You can enable HTTP/2 connectivity for an individual Ingress Controller or for the entire cluster.
+You can either enable HTTP/2 connectivity for an individual Ingress Controller or for the entire cluster.
```

Comment:
To elaborate on this a little, you can specify an explicit true/false preference on an IngressController, and that takes precedence over any true/false annotation on the cluster config.  Thus configuring HTTP/2 on the IngressController and configuring it on the cluster config are not mutually exclusive options, contrary to what the "either" could imply.

Reply:
Added:

> [NOTE]
====
If you enable HTTP/2 connectivity for an individual Ingress Controller and for the entire cluster, the HTTP/2 configuration for the Ingress Controller takes precedence over the HTTP/2 configuration for the cluster. 
====